# Plan, fleet, parameters for the VNS planner

# Plan name (unique)
string name

# Flight window
time[2] flight_window
